src/classes/OPP_OpportunityContactRoles_TDTM.cls

```diff
@@ -473,13 +495,33 @@ public without sharing class OPP_OpportunityContactRoles_TDTM extends TDTM_Runna
     * @param fieldName The name of the opportunity field.
     * @return OpportunityContactRole The newly created OCR.
     ********************************************************************************************************/
-    private OpportunityContactRole getOCR(Opportunity opp, string fieldName) {
+    private OpportunityContactRole getOCR(Opportunity opp, String fieldName) {
+        OpportunityContactRole ocr = new OpportunityContactRole();
```

Comment:
rather than duplicating this whole method, why not just have getOCR(opp, fieldname) call 
getOCR(opp, fieldname, null), so there is only one implementation.

Reply:
Glad to see this comment. I've updated the code accordingly.